packages/mdc-tabs/README.md

```diff
@@ -0,0 +1,414 @@
+       class="mdc-tab mdc-tab--active" href="#panel-1">Item One</a>
+    <a role="tab" aria-controls="panel-2"
+       class="mdc-tab" href="#panel-2">Item Two</a>
+    <a role="tab" aria-controls="panel-3"
+       class="mdc-tab" href="#panel-3">Item Three</a>
+    <span class="mdc-tab-bar__indicator"></span>
+  </nav>
+</section>
+<section>
+  <div class="panels">
+    <p class="panel active" id="panel-1" role="tabpanel" aria-hidden="false">Item One</p>
+    <p class="panel" id="panel-2" role="tabpanel" aria-hidden="true">Item Two</p>
+    <p class="panel" id="panel-3" role="tabpanel" aria-hidden="true">Item Three</p>
+  </div>
+  <div class="dots">
+    <a class="dot active" data-trigger="panel-1" href="#panel-1"></a>
+    <a class="dot" data-trigger="panel-2" href="#panel-2"></a>
+    <a class="dot" data-trigger="panel-3" href="#panel-3"></a>
+  </div>
+</section>
+```
+
+#### Script:
+```js
+var dynamicTabBar = window.dynamicTabBar = new mdc.tabBar.MDCTabBar(document.querySelector('#dynamic-tab-bar'));
+var dots = document.querySelector('.dots');
+var panels = document.querySelector('.panels');
+
+dynamicTabBar.preventDefaultOnClick = true;
+
+function updateDot(index) {
+  var activeDot = dots.querySelector('.dot.active');
+  if (activeDot) {
+    activeDot.classList.remove('active');
+  }
+  var newActiveDot = dots.querySelector('.dot:nth-child(' + (index + 1) + ')');
+  if (newActiveDot) {
+    newActiveDot.classList.add('active');
+  }
+}
+
+function updatePanel(index) {
+  var activePanel = panels.querySelector('.panel.active');
+  if (activePanel) {
+    activePanel.classList.remove('active');
+  }
+  var newActivePanel = panels.querySelector('.panel:nth-child(' + (index + 1) + ')');
+  if (newActivePanel) {
+    newActivePanel.classList.add('active');
+  }
+}
+
+dynamicTabs.root_.addEventListener('MDCTabBar:change', function ({detail: tabs}) {
+  var nthChildIndex = tabs.activeTabIndex;
+
+  updatePanel(nthChildIndex);
+  updateDot(nthChildIndex);
+});
+
+dots.addEventListener('click', function (evt) {
+  if (!evt.target.classList.contains('dot')) {
+    return;
+  }
+
+  evt.preventDefault();
+
+  var dotIndex = [].slice.call(dots.querySelectorAll('.dot')).indexOf(evt.target);
+
+  if (dotIndex >= 0) {
+    dynamicTabBar.foundation_.switchToTabAtIndex(dotIndex, false);
+  }
+  updatePanel(dotIndex);
+  updateDot(dotIndex);
+})
+```
+
+### Using the CSS-Only Component
+
+`mdc-tab-bar` ships with css for styling a tab layout according to the Material
+Design spec. To use CSS only tab bars, simply use the available class names.
+Note the available `mdc-tab--active` modifier class. This is used to denote the
+currently active tab.
+
+```html
+<nav class="mdc-tab-bar">
+  <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+  <a class="mdc-tab" href="#two">Item Two</a>
+  <a class="mdc-tab" href="#three">Three</a>
+  <span class="mdc-tab-bar__indicator"></span>
+</nav>
+```
+
+### Using the JavaScript Component
```

Comment:
We should, at some point in this section, talk about `tabFactory` as a way of controlling how `MDCTabBar` instantiates its `MDCTab` components.